tcn_hpl/models/ptg_module.py

```diff
@@ -100,12 +100,13 @@ def __init__(
 
         # Get Action Names
         mapping_file = f"{self.hparams.data_dir}/{mapping_file_name}"
-        file_ptr = open(mapping_file, "r")
-        actions = file_ptr.read().split("\n")[:-1]
-        file_ptr.close()
         actions_dict = dict()
-        for a in actions:
-            actions_dict[a.split()[1]] = int(a.split()[0])
+        with open(mapping_file, "r") as file_ptr:
+            actions = file_ptr.readlines()
+            actions = [a.strip() for a in actions]  # drop leading/trailing whitespace
+            for a in actions:
+                parts = a.split(" ")
```

Comment:
Actually, this is a little different than the previous version. The previous `.split()` with no arguments would split on any amount of whitespace between things, where as this will split on all spaces, even if there are multiple
![image](https://github.com/user-attachments/assets/ffa783b7-6b3c-4962-b68c-ef3b75beb5b4)
May want to prefer to just use the no-argument version
```suggestion
                parts = a.split()
```

Reply:
agreed - I added a comment too